src/pages/Dashboard.js

```diff
@@ -31,7 +31,13 @@ const yourFamilyRows = ({ maritalStatus, children }) => {
 }
 
 const yourIncomeRows = ({ income }) => {
```

Comment:
So you _could_ do (since you know everything is nested inside of the "income" key). 
I would generally prefer that, but this data is pretty changeable.

```suggestion
const yourIncomeRows = ({ income: { employerName, year, box12, box14, box22 } = {} }) => { 
return [
    { key: 'Employer Name', value: employerName },
    { key: 'Year', value: year },
    ...
  ]
  }
```

Reply:
I agree that I like this style better. Is the `= {}` a default value that's passed to the function?

Reply:
Yeah, if there's no value passed in (ie, you call `yourIncomeRows()`), then you default `income` to an empty object.

Otherwise, `income` will default to `undefined`, and trying to access `income.employerName` will cause runtime errors.